src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="main">
+      <div class="calendar calendar--month-length--29">
+        <div class="calendar__day calendar__day--start-day--sun"></div>
```

Comment:
It looks not good to use two modifiers together. You can rewrite it
```suggestion
      <div class="calendar calendar--month-length-29">
        <div class="calendar__day calendar__day--start-day-sun"></div>
```